### DISCUSSION TITLE
Reviews statistic is incorrect (equals to zero)

### DISCUSSION THREAD

**Discussion by Andrej123456789** [anuraghazra/github-readme-stats#3237] [Q&A] [answered]
### Describe the bug

My reviews statistic is incorrect and therefore my rank is wrong. I counted myself that I have ± 12 review throughout my existence on GitHub.

**NOTE**: I waited for 24 hours
And also, can I remove &show=reviews when we fix this and will rank be same?

### Expected behavior

To have 12 reviews and B+ rank

### Screenshots / Live demo link

![image](https://github.com/anuraghazra/github-readme-stats/assets/83548580/32318119-e37a-4956-b0ee-c8866ae659bb)

[![Anurag's GitHub stats](https://github-readme-stats-andrej123456789.vercel.app/api?username=Andrej123456789&theme=gruvbox&include_all_commits=true&show=reviews)](https://github.com/anuraghazra/github-readme-stats)

**Comment by qwerty541** [comment, marked as answer]
Hey, @Andrej123456789! Looks like it is not an error from our side, it is exactly what GitHub API returns, i tested it using GraphQL Explorer, you can check on the following screenshot.

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/011f4cdf-2b74-415e-b3e2-328ba587298f)

I guess that your reviews was made inside private repositories. As specified in documentation by default stats card show only your public contributions.

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/b39daa30-c8cf-4998-a269-52d9e884fd34)

**Reply by Andrej123456789** [reply]
Reviews are mostly made in [ReactOS](https://github.com/reactos/reactos) repository
I am using my own deployed instance

**Reply by qwerty541** [reply]
@Andrej123456789 Unfortunately i have no idea about reasons of such behavior, i see in your profile that you really have some reviews. As i previously noticed the card shows data fetched from GitHub API, maybe there is some error from their side. Their documentation tells that property `totalPullRequestReviewContributions` inside `ContributionsCollection` object should show how many pull request reviews the user left.

https://docs.github.com/en/graphql/reference/objects#contributionscollection

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/3c01dabf-ee56-4af4-9c57-b84d68fd1450)

@rickstaa May be you have some ideas about what we can do about that?

**Reply by rickstaa** [reply]
@qwerty541 I suspect @Andrej123456789 did not use the correct permissions when creating the PAT for your instance. @Andrej123456789, you can check this by running the following code inside the [GitHub graphql explorer](https://docs.github.com/en/graphql/overview/explorer):

```graphql
query userInfo($login: String!) {
  user(login: $login) {
    name
    login
    contributionsCollection {
      totalCommitContributions
      totalPullRequestReviewContributions
    }
  }
}
```

```graphql
{"login": "Andrej123456789"}
```

Since I don't have access to your private repositories, I get 0 reviews. You, however, should see all reviews when you log in. If you do, your PAT token needs to be corrected. If you do not, you should contact [GitHub support](https://support.github.com/).

![image](https://github.com/anuraghazra/github-readme-stats/assets/17570430/ea4f2af0-9ea7-475e-9222-fd9c894c297a)

**Reply by qwerty541** [reply]
@rickstaa Thanks for the answer. The problem unlikely connected to PATs permissions in my opinion since @Andrej123456789 has reviews in public repositories like https://github.com/ringwormGO-organization/ringOS/pull/32 and https://github.com/reactos/reactos/pull/4375. But it might still be worth checking. @Andrej123456789 This is how your personal access token should be configured:

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/c9b38ef1-68a8-4252-9efc-668409645fbe)

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/44b30b15-17c6-4790-9586-558744d54cf6)

**Reply by Andrej123456789** [reply]
I made new token with following permissions. I'll get you updated for 24 hours or less

**Reply by rickstaa** [reply]
@Andrej123456789 great! Did you also check the response from the GraphQL explorer? We use that data to create the card.

**Reply by Andrej123456789** [reply]
![image](https://github.com/anuraghazra/github-readme-stats/assets/83548580/ff524d1f-6674-4116-a57d-0d86f0df151b)

**Reply by rickstaa** [reply]
@Andrej123456789 Since you are logged in, you receive all public and private data. If you know you did reviews, it is an issue at GitHub's side. I would advise you to contact the [GitHub support](https://support.github.com/).

**Reply by Andrej123456789** [reply]
I'll wait 24 hours and then contact GitHub support
Thanks for help

**Reply by Andrej123456789** [reply]
Token is working which can be seen by updated language stats, so it's GitHub issue
